RELEASE.md

```diff
@@ -27,34 +27,91 @@ $ npm --version
 
 ## Branching
 
-This is the example branching/merging for release channels.
+If you're planning to release a stable/bugfix version _and_ a beta, make sure to release the beta _after_ the stable version.
 
 ```sh
-# Get to known good state.
-git checkout master
-git reset --hard origin/master
-git checkout beta
-git reset --hard origin/beta
-git checkout release
-git reset --hard origin/release
-
-# Prep the stable release
-git checkout release
-git merge beta
+# Fetch changes from GitHub
+git fetch origin
+```
+
+Once you're done following these instructions make sure that you pushed all your branches back to GitHub.
```

Comment:
should we be explicit as to which branches, `git pull --all` would push WIP local branches as well.